torch/csrc/jit/script/lexer.h

```diff
@@ -132,7 +134,7 @@ struct SharedParserData {
         {}, // reserve a level for unary not
         {'<', '>', TK_EQ, TK_LE, TK_GE, TK_NE},
         {'+', '-'},
-        {'*', '/'},
+        {'*', '/', TK_MATMUL, TK_POW},
```

Comment:
I think this will put multiplication and power at the same precedence level, which won't be correct. For example `2 * 2 ** 2` will effectively parse as `(2 * 2) ** 2` instead of `2 * (2 ** 2)`. You need to add extra brackets below this level and put `POW` there